stix_shifter_modules/guardium/stix_transmission/guard_utils.py

```diff
@@ -113,11 +114,28 @@ def handle_report(self, report_name, params, index_from, fetch_size):
         # -------------------------------------------------------------------------------
         results = ""
         #context().logger.debug('-------------------  ' + report_name + ' ----------------------')
-        params_set = {"reportName":"{0}".format(report_name), "indexFrom": "{0}".format(index_from), "fetchSize": "{0}".format(fetch_size), "reportParameter":params}
+        params_set = {"reportName":"{0}".format(report_name), "indexFrom": "{0}".format(index_from), "fetchSize": "{0}".format(fetch_size), "reportParameter":params, "inputTZ":"GMT"}
         json_dump = json.dumps(params_set)
         rest_data = str(json.loads(json_dump))
-
-        response = requests.post(self.url+self.report_target, data=rest_data, headers=self.headers, verify=False)
+        # For compatibility with Guardium - inputTZ parameter was aded after v11.3 so in case it does not exist execute the query without it
+        for x in range(2):
```

Comment:
not sure if this is the right way. you can just send the request with inputTZ first and check the response if error occurs update the the param and then send the request again. I would remove the for loop for this case

Reply:
This is what I actually do.
The performs the second round only if error code is 27